include/aikido/io/util.hpp

```diff
@@ -0,0 +1,22 @@
+#ifndef AIKIDO_IO_UTIL_HPP_
+#define AIKIDO_IO_UTIL_HPP_
+
+#include <aikido/io/CatkinResourceRetriever.hpp>
```

Comment:
Nit: Our convention is to:
(1) use `"..."` for including AIKIDO headers
(2) include headers in order (in header file): (a) system headers (i.e., STL), (2) 3rd party libraries, (3) AIKIDO headers.

So this should be:
```cpp
#include <dart/dart.hpp>
#include <dart/utils/urdf/DartLoader.hpp>
#include "aikido/io/CatkinResourceRetriever.hpp"
```

Reply:
I'll change it but could you tell me if there is a specific reason for the header include order? Does it have an advantage to do it this way?

Reply:
This header ordering generally follows the Google C++ style convention:
https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes